CAM Platform/api-cost-centers.md

```diff
@@ -0,0 +1,337 @@
+{{{
+"title": "Cost Centers API",
+"date": "12-19-2018",
+"author": "Julio Castanar",
+"attachments": [],
+"contentIsHTML": false,
+"keywords": ["api", "Cost center"]
+}}}
+
+**Manage Cost Centers**
+
+| Resource | Description |
+|----------|-------------|
+| [POST /services/costcenters](#post-servicescostcenters) | Creates a Cost Center. |
+| [GET /services/costcenters/{costcenter_id}](#get-servicescostcenters) |  Fetches an existing Cost Center. |
+| [PUT /services/costcenters/{costcenter_id}](#put-servicescostcenters) | Updates an existing Cost Center.|
+| [DELETE /services/costcenters/{costcenter_id}](#delete-servicescostcenters) | Deletes an existing Cost Center.|
+
+## POST /services/costcenters
+
+Creates a new Cost Center and gets the created Cost Center
+
+### URL
+
+#### Structure
+```
+[POST] /services/costcenters
+```
+#### Example
+```
+[POST] https://cam.ctl.io/services/costcenters
+```
+### Request
+
+#### Headers
+```
+Content-Type: application/json
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
+#### URI Parameters
+
+* None
+
+#### Request body parameters
+
+| Name | Type |Description | Req. |
+|------|------|------------|------|
+| name | string | Cost Center name. | Yes  |
+| organization | string | Name of the organization with access to the Cost Center. | Yes |
+| schema | string | Cost Center schema. Always “http://elasticbox.net/schemas/costcenter” | Yes  |
+| members | array | Members' id list of the Cost Center as Administrators. | No |
+
+```
+{
+    "name": "Cost Center example",
+    "organization": "elasticboxdemo",
+    "schema": "http://elasticbox.net/schemas/costcenter",
+    "members":[{"id":"elasticboxdemo"}]
```

Comment:
The documents in this array require 3 parameters: `id`, `role` and `type`. `role` will always be `administrator` and `type` can be `workpace` or `ldap`. `id` can then be a workspace ID or a ldap group DN.